src/Backend/Modules/Faq/Actions/Edit.php

```diff
@@ -152,7 +151,11 @@ private function validateForm(): void
 
     private function loadDeleteForm(): void
     {
-        $deleteForm = $this->createForm(FaqDeleteType::class, ['id' => $this->record['id']]);
+        $deleteForm = $this->createForm(
+            DeleteType::class,
```

Comment:
it is there, because I tested it -> https://github.com/jonasdekeukelaere/forkcms/blob/90a8e1a9a0cd8eec0ad9f543edaac4d8cd8c0aa0/src/Backend/Modules/Faq/Actions/Edit.php#L17